src/utils/hooks/useThemeContext.js

```diff
@@ -0,0 +1,7 @@
+import { useContext, createContext } from 'react'
```

Comment:
Just call this `useTheme`, and we can use it everywhere we'd normally use MUI's `useTheme`.